packages/keystone/cache.js

```diff
@@ -29,12 +29,25 @@
 const express = require('express')
 const { get, set } = require('lodash')
 
+const conf = require('@condo/config')
+
+const { getLogger } = require('./logging')
+
+const logger = getLogger('cache-l1')
+
+const ENABLE_CACHE_LOGGING = !!conf.ENABLE_CACHE_LOGGING
+
+
 class KeystoneCacheMiddleware {
 
     constructor (keystone) {
         this.cache = {}
 
-        initCache(keystone, this.cache)
+        this.hits = 0
+        this.requests = 0
+        this.hasRequestId = 0
+
+        initCache(keystone, this)
```

Comment:
Why not to create just StateObject instead of this?

Reply:
?

Reply:
Because you also need to transfer cache itself

Reply:
So, cache and state is located in the same class

I can make a `typeof` assertion in initCache function, but dont really know if it is going to help. This script is small